cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id="running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines"]
+= Running workloads and buildah as user namespaces on {pipelines-title}
+include::_attributes/common-attributes.adoc[]
+:context: running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines
+
+toc::[]
+
+Running {pipelines-shortname} as root on a container can expose the host and container processes to other resources. In some cases, to reduce this type of exposure, you can run some workloads in a user namespace instead. In other cases, you might still need to run the workload as root on the container but still be secure on the host.
```

Comment:
It should either be `Running {prod} as a root on a container...` or `Running {prod} as roots on a container...`

Reply:
Hi, Jake. Thanks for the suggestion. The original phrasing, `running x as root`, sounds right in Linux parlance and in the context of Pipelines. See `modules/op-running-as-root-user-namespace.adoc`, for example.